mirrord/kube/src/api.rs

```diff
@@ -14,15 +14,10 @@ use tracing::{error, info};
 use crate::error::{KubeApiError, Result};
 
 pub mod container;
-#[cfg(feature = "env_guard")]
-mod env_guard;
-#[cfg(feature = "env_guard")]
-pub use env_guard::MIRRORD_GUARDED_ENVS;
-
 pub mod kubernetes;
 mod runtime;
 
-static CONNECTION_CHANNEL_SIZE: usize = 1000;
+const CONNECTION_CHANNEL_SIZE: usize = 1000;
```

Comment:
Warning: missing docs.

```suggestion
/// Size of the channel used in [`wrap_raw_connection`] for [`ClientMessage`]s.
const CONNECTION_CHANNEL_SIZE: usize = 1000;
```